### PR TITLE
docs(perf): Add docs for ky integration & full example

### PR DESCRIPTION
**Pull request by jack828**
### Description

Added because I just setup this integration and I feel the community could benefit.
https://github.com/invertase/react-native-firebase/discussions/5695

### Release Summary

 Add docs for ky intergration

### Checklist

- I read the [Contributor Guide](../CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
  - [x] Yes
- My change supports the following platforms;
  - [x] (N/A) `Android`
  - [x] (N/A) `iOS`
- My change includes tests;
  - [x] (N/A) `e2e` tests added or updated in `packages/\*\*/e2e`
  - [x] (N/A) `jest` tests added or updated in `packages/\*\*/__tests__`
- [x] (N/A) I have updated TypeScript types that are affected by my change.
- This is a breaking change;
  - [ ] Yes
  - [x] No

### Test Plan

(N/A) 

---
🔥

### CODE REVIEW

File: docs/perf/axios-integration.md
Comment (mikehardy) [inline docs/perf/axios-integration.md:4]: I was suspicious of this because normally when you/we slice a page in we need to also patch up both the next/prev pointers and find the next page then fix up it's prev, but no! Axios was the last page, and this will be the new last page, so this is perfect

**Review by mikehardy — APPROVED** [review]
This looks fantastic - Not just useful as an idea + great content, but I haven't seen a non-trivial docs PR recently (including my own) that did not need a spellcheck additionor markdown lint fixup before merge but it looks like you must have even worked through that - huge +1 from me as a maintainer !

Thanks for posting this

**Comment by jack828** [comment]
Thank you! I wanted to make a painless contribution 😁 thanks for the rapid merge too!! 🔥

**Pull request merged**
